generator.py

```diff
@@ -0,0 +1,40 @@
+import random
+import melody
+
+
+random.seed()
+
+
+def generate(note_threads_number, accord_threads_number, length):
+    """
+    this function produces melody which contains set of melodies: common, and accord-melodies
+    :param note_threads_number: (int)how many flows of common melodies do we want
+    :param accord_threads_number: (int)how many flows of accord-melodies do we want
+    :param length: (int)how many notes should be in the melody
+    :return: (list of Melody) all the melodies which music contains
+    """
+    seq_seq = []
+
+    j = 0
+    while j < note_threads_number:
+        i = 0
+        this_mel = melody.Melody(0)
+        while i < length:
```

Comment:
Same goes here. `for _ in range(i, track_length)` is better

Reply:
fixed